templates/dotnet/template/src/Main.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text;
+using Nitric.Faas;
+
+namespace NitricTemplate
+{
+    class HelloWorld : INitricFunction

```

Comment:
Change the names across the board to something more usable. E.g. Handler instead of HelloWorld. namespace should be Function or FunctionHandler (something that's reasonable without needing to be changed).